packages/conf-radare2/conf-radare2.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+name: "conf-radare2"
+version: "0.1"
+maintainer: "Ivan Gotovchits <ivg@ieee.org>"
+authors: "BAP Team"
+homepage: "https://github.com/BinaryAnalysisPlatform/bap/"
+bug-reports: "https://github.com/BinaryAnalysisPlatform/bap/issues"
+dev-repo: "git://github.com/BinaryAnalysisPlatform/bap/"
+license: "MIT"
+build: [
+  ["ocaml" "unix.cma" "find-radare2.ml"]
+]
+depends: [
+   "ocaml"
```

Comment:
if I remember correctly the `{| .. |}` type of string came in OCaml 4.02 so I think this should be constrained.

Reply:
done! :)